howso.amlg

```diff
@@ -496,6 +496,9 @@
 
 			;a value used to limit the allocations of each usage of (call_sandboxed), 0 = no limit
 			!sandboxedMemoryLimit 100000
+			
+			;a value used to limit the depth of opcode operations of each usage of (call_sandboxed)
+			!sandboxedOpcodeDepthLimit 50
```

Comment:
need to declare this label uptop where others are declared, around line 167 or so